### DISCUSSION TITLE
Question - How to import github project to eclipse ?

### DISCUSSION THREAD

**Discussion by BaptisteFournie** [Estia-advanced-programming/pandora-public#77] [Q&A] [answered]
I don't understand how to import my github project to eclipse ? 
I think it's hard to find the way to import in the assigment.

**Comment by WilliamDelamare** [comment, marked as answer]
Some students can get confused by the reading flow of the Git and Eclipse wiki parts.

* **Clone the project**: read the first part of the `git` wiki section (https://github.com/Estia-advanced-programming/pandora-public/wiki/Tutorial-%C2%BB-Git-Client)
* **Import the project**: read the `Eclipse` wiki section (https://github.com/Estia-advanced-programming/pandora-public/wiki/Tutorial-%C2%BB-Eclipse)
* **Use Git commands in eclipse**: read the second part of the `git` wiki section (https://github.com/Estia-advanced-programming/pandora-public/wiki/Tutorial-%C2%BB-Git-Client)
